include/sharg/detail/format_base.hpp

```diff
@@ -183,6 +184,63 @@ class format_base
 
         return tmp;
     }
+
+    /*!\brief Returns the default message for the help page.
+     * \tparam option_type The type of the option.
+     * \tparam default_type  The type of the default value.
+     * \param[in] option The option to get the default message for.
+     * \param[in] value The default value to get the default message for.
+     * \returns The default message for the help page (" Default: <default-value>. ").
+     * \details
+     * `value` is either `config.default_message`, or the same as `option`.
+     * If the `option_type` is a std::string or std::filesystem::path, the value is quoted.
+     * If the `option_type` is a container of std::string or std::filesystem::path, each individual value is quoted;
+     * if a `config.default_message` is provided, it will not be quoted.
+     */
+    template <typename option_type, typename default_type>
+    static std::string get_default_message(option_type const & SHARG_DOXYGEN_ONLY(option), default_type const & value)
```

Comment:
FYI: This doesn't need to be `static` because it is already `inline` (it is inline, because it's a template).

Reply:
I take this back. This is not a free function but a member function. This should be static!